app/src/main/java/org/sopt/collaboration/campuspick/feature/search/FilterBottomSheet.kt

```diff
@@ -0,0 +1,143 @@
+import org.sopt.collaboration.campuspick.core.ui.lifecycle.LaunchedEffectWithLifecycle
+import org.sopt.collaboration.campuspick.domain.model.DeadLine
+import org.sopt.collaboration.campuspick.domain.model.DeadLine.Companion.fromLabel
+import org.sopt.collaboration.campuspick.domain.model.FilterSection
+import org.sopt.collaboration.campuspick.domain.model.Location
+import org.sopt.collaboration.campuspick.domain.model.Location.Companion.fromLabel
+import org.sopt.collaboration.campuspick.domain.model.PreferDay
+import org.sopt.collaboration.campuspick.domain.model.PreferDay.Companion.fromLabel
+import org.sopt.collaboration.campuspick.feature.search.component.SearchKeyword
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun FilterBottomSheet(
+    showFilterBottomSheet: Boolean,
+    onDismiss: (Boolean) -> Unit,
+    bottomSheetDeadLineSelected: String,
+    updateSelectedDeadLine: (DeadLine) -> Unit,
+    bottomSheetLocationSelected: String,
+    updateSelectedLocation: (Location) -> Unit,
+    bottomSheetPreferDaySelected: String,
+    updateSelectedPreferDay: (PreferDay) -> Unit,
+) {
+    val coroutineScope = rememberCoroutineScope()
+    val bottomSheetState = rememberModalBottomSheetState(
+        skipPartiallyExpanded = true
+    )
+
+    LaunchedEffectWithLifecycle {
+        if (showFilterBottomSheet) {
+            coroutineScope.launch {
+                bottomSheetState.show()
+            }
+        }
+    }
+
+    if (showFilterBottomSheet) {
+        ModalBottomSheet(
+            onDismissRequest = {
+                coroutineScope.launch {
+                    bottomSheetState.hide()
+                }.invokeOnCompletion {
+                    onDismiss(false)
+                }
+            },
+            sheetState = bottomSheetState,
+            dragHandle = {
+                Column(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .background(CampuspickTheme.colors.White),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                ) {
+                    BottomSheetDefaults.DragHandle()
+                }
+            }
+        ) {
+            Box(
+                modifier = Modifier
+                    .background(CampuspickTheme.colors.White),
+                contentAlignment = Alignment.Center,
+            ) {
+                Column(
+                    verticalArrangement = Arrangement.Top,
+                    modifier = Modifier
+                        .padding(horizontal = 15.dp)
+                        .fillMaxWidth()
+                        .background(CampuspickTheme.colors.White),
+                ) {
+                    val filterSection = listOf<FilterSection>(
+                        FilterSection(
+                            title = "마감 기한",
+                            keywords = DeadLine.entries.mapNotNull { it.label.takeIf { label -> label.isNotBlank() } },
```

Comment:
EMPTY를 제외하고 모든 목록을 가져오는 거라면, `label.isNotBlank()` 대신 filter를 활용할 수 있을 것 같습니다.
```suggestion
                            keywords = DeadLine.entries.filter { it != Deadline.EMPTY },
```

Reply:
enum 전문가 👍
저는 당연히 label의 string으로만 비교하려 했습니다.ㅎㅎ